org.gnome.Ptyxis.json

```diff
@@ -0,0 +1,96 @@
+{
+    "app-id" : "org.gnome.Ptyxis",
```

Comment:
This is not hosted in `GNOME` on the gnome gitlab, so I'm not sure `org.gnome` is allowed.

Reply:
It's already in GNOME's nightly fwiw. Mostly a matter of process at this point.

Reply:
Filed https://gitlab.gnome.org/Infrastructure/Infrastructure/-/issues/1362

Reply:
This is resolved upstream now